pilot/copytool/rucio_api.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python
```

Comment:
Tomas, why did you clone the copytool?

Reply:
Ah, sorry, that was just my local testing version.